README.md

```diff
@@ -21,6 +21,50 @@ It is executed via an init container, acquiring certificates on pod start.
   see [docs](https://kubernetes.io/docs/tasks/configure-pod-container/share-process-namespace/).
   - This will require configuring kubernetes to grant the `SYS_PTRACE` capability to the container.
 
+#### Generating Certificates
```

Comment:
Reading this the first time, I think this section would be better placed under a `Setup certificate authority` heading.

Also I think it would be nice to setup a common base for the CA too, and this part would sit in there. But until we get there, these kind of updates are more than welcome.

Reply:
Yeah that was a another stumbling block when working on this. I could split this into two files linked in the readme? Then when we do split the repo this is slightly easier?

Reply:
Indeed, that's a great idea